### ISSUE TITLE
Show Playlist + Replace play queue with selected songs playlist

### ISSUE THREAD

**Issue by ghost** [vdesabou/alfred-spotify-mini-player#14]
Suggestion 1:

In the mini player where you list the playlists that the currently playing track is part of, would it be possible to add an "show playlist in Spoitfy" option to jump you to the Spotify app with the selected playlist displayed?

Could also be added to the normal playlist list items?

Suggestion 2:

When selecting songs to play from playlists it just adds the selected song to the Play Queue and plays it, after the selected song the play queue continues with whatever you had in your play list previously. Would it be possible that when you select a song from a playlist the playlists becomes the now playing list and the selected song gets played. I imagine you would need to "launch the playlist" then set the now playing song?

Basically it's caught me out a few times when listening to Artist A, then wanting to switch to a specific song from my Artist B playlist, but after that song Artist A would come back on. I expected that when selecting a song from a playlist that playlist overwrites the play queue and the selected song plays, as that is what happens in the Spotify app.
